isites_migration/views.py

```diff
@@ -29,7 +29,11 @@ def index(request):
         )
         return redirect('isites_migration:index')
 
-    processes = Process.objects.filter(name='isites_migration.jobs.migrate_files').order_by('-date_created')
+    processes = Process.objects.filter(
+        name='isites_migration.jobs.migrate_files',
+        details__at_canvas_course_id=canvas_course_id
```

Comment:
This is django-pgjson query syntax.

http://djangonauts.github.io/django-pgjson/#_querying